frappe/public/js/frappe/model/model.js

```diff
@@ -318,11 +318,13 @@ $.extend(frappe.model, {
 	},
 
 	can_import: function(doctype, frm) {
+		if (!doctype.allow_import) return false;
+
```

Comment:
```suggestion
	can_import: function(doctype, frm, meta=null) {
		if (meta and !meta.allow_import) return false;
```
instead of using doctype as meta, pass meta separately to keep it explicit.
Also, this will not break `frappe.model.can_import` (for anyone who might have used it in a different application) and it will be consistent with other `can_*` utility functions in this file